### PR TITLE
Api appointments

### PR DESCRIPTION
**Pull request by basicem**

### CODE REVIEW

File: controllers/appointment.controller.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const appointmentService = require("../services/appointment.services");
+const authService = require("../services/auth.services");
+const { appointmentSchema } = require("../schemas/appointment");
+const { resolveError } = require("../helpers/controllers");
+
+const create = async (req, res) => {
+  try {
+    const authorizationHeader = req.headers.authorization;
+    const token = authorizationHeader.split(" ")[1];
+    const { id } = await authService.verifyToken(token);
```
Comment (amarbasic) [inline controllers/appointment.controller.js]: Your current user id is in the req object https://github.com/basicem/medica-api/pull/11/files#diff-f30160aefa12ca4840ebd31571a3ba802d37dbd8b00cffa0a0a0387adcf738f6L12

File: controllers/appointment.controller.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    const data = { ...req.body, doctorId: id };
+    const value = await appointmentSchema.validateAsync(data);
+    const appointment = await appointmentService.createAppointment(value);
+
+    // send the mail
+    const toEmail = process.env.RECIPIENT_EMAIL;
```
Comment (amarbasic) [inline controllers/appointment.controller.js]: toEmail should be patient email?

File: controllers/appointment.controller.js (hunk 1 above)
Comment (basicem) [reply controllers/appointment.controller.js]: ?

Comment (basicem) [reply controllers/appointment.controller.js]: fixed it

**Pull request merged**